projects/addon-mobile/components/sheet-dialog/sheet-dialog.options.ts

```diff
@@ -12,6 +12,7 @@ export interface TuiSheetDialogOptions<I = undefined> {
     readonly offset: number;
     readonly stops: readonly string[];
     readonly bar: boolean;
+    readonly fullsize: boolean;
```

Comment:
let's name this option `fullscreen`, like the size in the dialogs and like in figma